open.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
```

Comment:
This could be put a single script and launched after options have been parsed.

Reply:
Can you please expand on this idea, I'm not sure I got it?

Reply:
For instance: gnu-koryavov --download
                        gnu-koryavov --open=<page_num>

Instead of: download .sh
                    open.sh

Reply:
I've moved `download.sh` and `open.sh` to `download()` and `open()` functions. I think, options parsing isn't my concern, but I did my best to simplify `download()` and `open()` calling.